packages/preact/README.md

```diff
@@ -111,11 +111,11 @@ function Person() {
 
 This way we'll bypass checking the virtual-dom and update the DOM property directly.
 
-## Utility Components and Hooks
+### Utility Components and Hooks
```

Comment:
This was nested a layer deeper as it's part of the integration, not equal & separate to it. You're not using it w/out the Preact integration, even if it exists on a subpath.